src/plugins/dashboard/public/application/lib/migrate_app_state.ts

```diff
@@ -106,9 +106,9 @@ export function migrateAppState(
     delete appState.uiState;
   }
 
-  appState.panels.forEach((panel) => {
-    panel.version = opensearchDashboardsVersion;
-  });
+  // appState.panels.forEach((panel) => {
```

Comment:
Lets either remove this if its not necessary or add a comment here with a tracking issue that mentions how this can be removed.